lib/interface_loader.js

```diff
@@ -44,6 +74,16 @@ let interfaceLoader = {
   },
 
   loadInterface(packageName, type, messageName) {
```

Comment:
We have the same logic in https://github.com/RobotWebTools/rclnodejs/blob/develop/index.js#L170, and we are going to hide the `requie` method, I suggest using this implementation to replace the one of `require` in `index.js`.

Reply:
Will log an issue later to do this change. Currently `require()` can do more than `loadInterface()` and there are test cases relying on it.